doc/guides/bluetooth/bluetooth-dev.rst

```diff
@@ -82,18 +82,33 @@ combination of two devices:
 Refer to :ref:`bluetooth_qemu_posix` for full instructions on how to build and
 run an application in this setup.
 
-Native POSIX with BabbleSim
-===========================
+Simulated nRF52 with BabbleSim
+==============================
 
 .. note::
    This is currently only available on GNU/Linux
 
-`BabbleSim`_ is a simulator of the physical layer of shared medium networks,
-and it can be used to develop BLE applications with Zephyr when combined with
-the Native POSIX target.
-When developing with BabbleSim, only :ref:`Combined builds
-<bluetooth-build-types>` are possible, since the whole Linux application is a
-single entity that communicates with the simulator.
+The :ref:`nrf52_bsim board <nrf52_bsim>`, is a simulated target board
+which emulates the necessary peripherals of a nrf52 SOC to be able to develop
+and test BLE applications.
+This board, uses:
+
+   * `BabbleSim`_ to simulate the nrf52 modem and the radio environment.
+   * The POSIX arch to emulate the processor.
+   * `Models of the nrf52 HW <https://github.com/BabbleSim/ext_NRF52_hw_models/>`_
+
+When you build your application targeting this board, you produce a native
```

Comment:
Can we have this somewhere shared between Native POSIX and nRF52 simulated?

Reply:
Any proposal?

Reply:
Yes, move it to the previous section and refer to that one from the present one

Reply:
At the same time you could clarify the difference between the arch and the board, with the arch being described in the previous section.

Reply:
Regarding the posix arch vs native_posix board, I think that is quite a digression for this section. Maybe I should just create another document for the architecture itself, separated from the board. But that would be another PR :)